### PR TITLE
[C-2857] Revert remove get blocknumber (#3802)"

### PR DESCRIPTION
**Pull request by dylanjeffers**
This reverts commit 233b585ef94bcc391aebea2273501eb943bca7a5.

### Description

This leads to an indexing skip otherwise.

### CODE REVIEW

**Review by nicoback2 — APPROVED** [review]

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-revert-get-blocknumber

**Review by isaacsolo — APPROVED** [review]

File: packages/web/src/components/collection/desktop/Artwork.tsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ export const Artwork = (props: ArtworkProps) => {
 
   useEffect(() => {
     // If there's a gradient, this is a smart collection. Just immediately call back
-    if (image || gradient || imageOverride || image === '') callback()
```
Comment (isaacsolo) [inline packages/web/src/components/collection/desktop/Artwork.tsx:65]: maybe this needed? since this is separate from ^

Comment (dylanjeffers) [reply packages/web/src/components/collection/desktop/Artwork.tsx:65]: oh wow yup thanks

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-revert-get-blocknumber

**Pull request merged**